### PR TITLE
Fix 'No resource for locale 'lang' found ' error

### LINKED ISSUE #3820: No resource for locale 'de' found
### Server Implementation

Paper

### Server Version

1.16.5

### Describe the bug

I have now changed the language in the config file from en to de but now I get this
WARN [PlotSquared/ClassLoaderCaptionProvider] No resource for locale 'de' found
warning message why does this come when I have specified it correctly?

### To Reproduce

 I downloaded the file and put it in the lang folder
https://s20.directupload.net/images/221002/49pg883v.png

### Expected behaviour

yes, what can I say, I expected that there would be no more errors. The language is in German, but this error message comes up
It would be nice if there were a PlotSquared version where there were no more errors

### Screenshots / Videos

_No response_

### Error log (if applicable)

https://athion.net/ISPaster/paste/view/87dd95ce75df45ffbbc3a8f0f77a1744

### Plot Debugpaste

https://s20.directupload.net/images/221002/b8bqpzlx.png

### PlotSquared Version

PlotSquared v6 6.10.0

### Checklist

- [X] I have included a Plot debugpaste.
- [X] I am using the newest build from https://www.spigotmc.org/resources/77506/ and the issue still persists.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by BlockyTheDev**
## Overview

Fixes #3820

## Description
Logs the message "No resource for locale" as info instead of a warning and reword the message a bit.

## Submitter Checklist

- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] New public fields and methods are annotated with `@since TODO`.
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dordsor21** [comment]
This should be a warning-level message at the minimum, and possibly an error to be honest. I think the message should also be something like `No resource for locale '{}' found in the plugin file, please ensure you have placed a messages_{}.json file in the 'lang' folder. You may be able to find completed translations at https://intellectualsites.crowdin.com/plotsquared`

**Comment by SirYwell** [comment]
> This should be a warning-level message at the minimum, and possibly an error to be honest. I think the message should also be something like `No resource for locale '{}' found in the plugin file, please ensure you have placed a messages_{}.json file in the 'lang' folder. You may be able to find completed translations at https://intellectualsites.crowdin.com/plotsquared`

It's definitely not an error, as it's totally fine to have message files other than the one for english. It's only worth to mention because automated patching of the message files will fall back to insert english messages (as there are no translations available) which need to be changed manually by the user. I still think info is totally fine.

**Comment by BlockyTheDev** [comment]
> > This should be a warning-level message at the minimum, and possibly an error to be honest. I think the message should also be something like `No resource for locale '{}' found in the plugin file, please ensure you have placed a messages_{}.json file in the 'lang' folder. You may be able to find completed translations at https://intellectualsites.crowdin.com/plotsquared`
> 
> It's definitely not an error, as it's totally fine to have message files other than the one for english. It's only worth to mention because automated patching of the message files will fall back to insert english messages (as there are no translations available) which need to be changed manually by the user. I still think info is totally fine.

I changed the wording a bit, so I hope that it's now a bit more clear to end users what to do. I think info is the better solution, because for example when I saw the message the first time I taught "What is going on." until I found out why the message is in the console.

File: Core/src/main/java/com/plotsquared/core/configuration/caption/load/ClassLoaderCaptionProvider.java [hunk 1]
```diff
@@ -53,7 +53,8 @@ final class ClassLoaderCaptionProvider implements DefaultCaptionProvider {
         try {
             final InputStream stream = this.classLoader.getResourceAsStream(url);
             if (stream == null) {
-                LOGGER.warn("No resource for locale '{}' found", locale);
+                LOGGER.info("No resource for locale '{}' found in the plugin file, please ensure you have placed the latest version of the file messages_{}.json in the 'lang' folder. You may be able to find completed translations at https://intellectualsites.crowdin.com/plotsquared",
```
Comment (SirYwell) [inline Core/src/main/java/com/plotsquared/core/configuration/caption/load/ClassLoaderCaptionProvider.java:56]: Please put a few line breaks here to make the code easier to read

Comment (BlockyTheDev) [reply Core/src/main/java/com/plotsquared/core/configuration/caption/load/ClassLoaderCaptionProvider.java]: Done.

**Review by SirYwell — APPROVED** [review]
Looks good.

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**